services/users/errors.go

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package users
+
+import "errors"
+
+var (
+	AcceptedDomainError  error = errors.New("the email provided does not belong to an accepted domain")
+	VerifyUserError      error = errors.New("could not update verify email field")
+	UserCountError       error = errors.New("could not get the total number of the users.")
+	InvalidPasswordError error = errors.New("invalid password")
```

Comment:
`InvalidPasswordError` and `ErrInvalidPassword` looks a bit similar to me. Can we treat raw `InvalidPasswordError` as `ErrInvalidPassword` with blank id?